Mathlib/Tactic/SeqFocus.lean

```diff
@@ -0,0 +1,43 @@
+/-
+Copyright (c) 2022 Jeremy Avigad. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Jeremy Avigad
+-/
+import Lean
+
+open Lean Elab Meta Tactic
+
+namespace Tactic
+
+/-- Assuming there are `n` goals, `mapTacs [t1, t2, ..., tn]` applies each `ti` to the respective
+goal and leaves the resulting subgoals. -/
+elab "mapTacs " "[" ts:tactic,* "]" : tactic => do
```

Comment:
```suggestion
/-- Assuming there are `n` goals, `map_tacs [t1, t2, ..., tn]` applies each `ti` to the respective
goal and leaves the resulting subgoals. -/
elab "map_tacs " "[" ts:tactic,* "]" : tactic => do
```
Tactics use snake case, because they are keywordish.